usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/basic/XWalkViewWithOnHideOnShowAsync.java

```diff
@@ -44,7 +44,7 @@ public final void onXWalkInitFailed() {
     public final void onXWalkInitCompleted() {
         StringBuffer mess = new StringBuffer();
         mess.append("Test Purpose: \n\n")
-        .append("Verifies XWalkView can hide and show.\n\n")
+        .append("Verifies XWalkView can hide and show.(Need to check this case in Android TV)\n\n")
```

Comment:
Right, the case does not distinguish the platforms. But the issue(XWALK-6120) only exists on Android TV. So add this checkpoint to remind the tester to check it on Android TV.